packages/tslint-config/tslint.json

```diff
@@ -15,6 +15,7 @@
     "jsx-boolean-value": false,
     "jsx-no-lambda": false,
     "jsx-no-multiline-js": false,
+    "no-default-export": true,
```

Comment:
👍

Reply:
This will give bunch of errors in other projects.. but that's a good thing!

Reply:
Yes, I will mark it as breaking